src/Components/FlashBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Banner, Sans } from "@artsy/palette"
+
+interface Props {
+  messageCode: string
```

Comment:
Could it make sense for this to be a TS enum? We don't need to handle arbitrary strings here yet (only using for email confirmation atm), so further specifying the types feels a bit safer.